File: model/src/main/kotlin/com/qzl/model/ucenter/response/JwtResult.java
package com.qzl.model.ucenter.response;


import com.qzl.common.model.response.ResponseResult;
import com.qzl.common.model.response.ResultCode;

/**
 * Created by mrt on 2018/5/21.
 */
public class JwtResult extends ResponseResult {
    public JwtResult(ResultCode resultCode, String jwt) {
        super(resultCode);
        this.jwt = jwt;
    }
    private String jwt;
}

File: model/src/main/kotlin/com/qzl/model/ucenter/response/JwtResult.kt
package com.qzl.model.ucenter.response


import com.qzl.common.model.response.ResponseResult
import com.qzl.common.model.response.ResultCode

/**
 * Created by mrt on 2018/5/21.
 */
class JwtResult(resultCode: ResultCode, private val jwt: String) : ResponseResult(resultCode)
